fe/fe-core/src/main/java/com/starrocks/lake/StarOSAgent.java

```diff
@@ -344,6 +354,7 @@ private void tryRemovePreviousWorker(long nodeId) {
         if (prevWorkerId < 0) {
             return;
         }
+        prepare();
```

Comment:
this is not needed since it's call by `addWorker`, prepare is already called in it